content/en/blog/2024-11-08-experience-helm-release-kubecon-na-24.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "Experience a Helm Release: Live at KubeCon + CloudNativeCon North America 2024!"
+slug: "experience-helm-release-kubecon-na-24"
+authorname: "Andrew Block"
+author: "@sabre1041"
+authorlink: "https://mattfarina.com"
```

Comment:
I think this is the wrong author link 😉

Reply:
you get all the credit that you can get!

Updated authorlink